proxy/src/test/java/com/wavefront/agent/listeners/ZipkinPortUnificationHandlerTest.java

```diff
@@ -0,0 +1,143 @@
+package com.wavefront.agent.listeners;
+
+import com.google.common.collect.ImmutableList;
+
+import com.wavefront.agent.handlers.MockReportableEntityHandlerFactory;
+import com.wavefront.agent.handlers.ReportableEntityHandler;
+
+import org.easymock.EasyMock;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.http.DefaultFullHttpRequest;
+import io.netty.handler.codec.http.FullHttpRequest;
+import io.netty.handler.codec.http.FullHttpResponse;
+import io.netty.handler.codec.http.HttpMethod;
+import io.netty.handler.codec.http.HttpVersion;
+import wavefront.report.Annotation;
+import wavefront.report.Span;
+import zipkin2.Endpoint;
+import zipkin2.codec.SpanBytesEncoder;
+
+import static org.easymock.EasyMock.createNiceMock;
+import static org.easymock.EasyMock.expectLastCall;
+import static org.easymock.EasyMock.replay;
+import static org.easymock.EasyMock.reset;
+import static org.easymock.EasyMock.verify;
+
+public class ZipkinPortUnificationHandlerTest {
+  private ReportableEntityHandler<Span> mockTraceHandler =
+      MockReportableEntityHandlerFactory.getMockTraceHandler();
+  private long startTime = System.currentTimeMillis();
+
+  @Test
+  public void testZipkinHandler() throws Exception {
+    ZipkinPortUnificationHandler handler = new ZipkinPortUnificationHandler("9411",
```

Comment:
Ok for now but let's fix this in the future - this port could be already used, causing tests to fail.